docs/google/engine.md

```diff
@@ -191,3 +191,51 @@ for b in range(num_circuits_in_batch):
      idx+=1
 ```
 
+
+## Downloading historical results
+
+Results from  previous computations are archived and can be downloaded later.
+In order to retrieve previous results, you will need the program id and the
+job id.  This can be retrieved from the job object when you run a sweep.
+Currently, getting the program and job ids can only be done through the
+`Engine` interface and not through the sampler interface.
+You can then use `get_program` and `get_job` to retrieve the results.
+See below for an example:
+
+```
+# Initialize the engine object
+engine = cirq.google.Engine(project_id='YOUR_PROJECT_ID')
```

Comment:
I would also mention that results are restricted to the project; only project collaborators are allowed to access the data (so people are not mislead this is a public database).

Reply:
Added this in the beginning of the section.